python/ray/data/_internal/planner/plan_read_op.py

```diff
@@ -78,8 +83,20 @@ def get_input_data(target_max_block_size) -> List[RefBundle]:
     )
 
     def do_read(blocks: Iterable[ReadTask], _: TaskContext) -> Iterable[Block]:
+        """Yield from read tasks, with retry logic upon transient read errors."""
         for read_task in blocks:
-            yield from read_task()
+            if read_task._metadata.input_files is not None:
+                read_files_name = ",".join(read_task._metadata.input_files)
```

Comment:
the input files list may be very large, and will make the error message too verbose.

Reply:
good point, updated to just use read fn name by default for all cases (since we have no control over the length of even one file name)